.github/workflows/build_and_test_upstream_ci_image.yml

```diff
@@ -32,3 +32,15 @@ jobs:
         run: |
           mkdir test_operator_work_dir output_logs
           podman run -it -v $PWD:/project/operator-test-playbooks -v ./Dockerfiles/ci/example-metadata-without-alm-annotations:/project/operator_dir -v ./output_logs:/project/output -v ./test_operator_work_dir:/project/test_operator_work_dir -e TEST_NAME=test_for_report_success_empty_alm_examples -e OPERATOR_DIR=/project/operator_dir -e WORK_DIR=/project/output -e OPERATOR_WORK_DIR=/project/test_operator_work_dir/ upstream_ci_image:latest
+      
+      - name: "Run the bundle validation test for a bundle that supports a very old OCP version --> negative test"
+        continue-on-error: false
+        run: |
+          mkdir test_operator_work_dir output_logs
+          podman run -it -v $PWD:/project/operator-test-playbooks -v ./Dockerfiles/ci/example-metadata-without-alm-annotations:/project/operator_dir -v ./output_logs:/project/output -v ./test_operator_work_dir:/project/test_operator_work_dir -e TEST_NAME=test_for_report_success_empty_alm_examples -e OPERATOR_DIR=/project/operator_dir -e WORK_DIR=/project/output -e OPERATOR_WORK_DIR=/project/test_operator_work_dir/ upstream_ci_image:latest
+      
+      - name: "Run the bundle validation test for a bundle with a syntactically incorrect OCP version range --> negative test"
+        continue-on-error: false
+        run: |
+          mkdir test_operator_work_dir output_logs
+          podman run -it -v $PWD:/project/operator-test-playbooks -v ./Dockerfiles/ci/example-metadata-without-alm-annotations:/project/operator_dir -v ./output_logs:/project/output -v ./test_operator_work_dir:/project/test_operator_work_dir -e TEST_NAME=test_for_report_success_empty_alm_examples -e OPERATOR_DIR=/project/operator_dir -e WORK_DIR=/project/output -e OPERATOR_WORK_DIR=/project/test_operator_work_dir/ upstream_ci_image:latest
```

Comment:
I am not sure what do you test here.
Cause I cannot se any checking for messages or for return code.

Using github actions as a testing frameworks is a no go.

We already wrote test with python unittest
https://github.com/redhat-openshift-ecosystem/operator-test-playbooks/blob/master/Dockerfiles/ci/run_tests.py

They are called implicitly cause the file is set as CMD in Dockerfile
https://github.com/redhat-openshift-ecosystem/operator-test-playbooks/blob/master/Dockerfiles/ci/Dockerfile#L14